web/modules/custom/profile_block/profile_block.module

```diff
@@ -0,0 +1,14 @@
+<?php
+ 
```

Comment:
Add the hook implementation doc here.

Reply:
changed here : https://github.com/guptahemant/insta-clone/pull/87/files#diff-cc935b75bd78138b41a1c08524b1faff962ebae85397bd3e55f0f768c7a7f6a9